content/docs/intro/concepts/programming-model.md

```diff
@@ -573,6 +573,7 @@ All resource constructors also accept an `options` argument which can provide th
 * [`parent`](#parent): establish a parent/child relationship between resources
 * [`protect`](#protect): prevent accidental deletion of a resource by marking it protected
 * [`provider`](#provider): pass an [explicitly configured provider](#explicit-provider-configuration), instead of using the default global provider
+* [`version`](#version): pass a provider plugin version that should be used when operating on a resource.
```

Comment:
Nit: I get why this is under `provider`, but the rest of the bullets are alphabetical. Should we keep the alphabetical ordering and move this below `transformations` (both here in the bullets and the linked section below)?

Reply:
Great point!